games/fheroes2/Portfile

```diff
@@ -7,6 +7,7 @@ PortGroup               cmake 1.1
 
 github.setup            ihhub fheroes2 0.9.4
 
+revision                1
```

Comment:
if it is only fixing build failures (and the change does not affect the outcome for the systems that did build correctly) then there is no need to increase the revision